content/en/docs/installation/helm.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Helm"
+linkTitle: "Helm"
+weight: 22
+type: "docs"
+---
+
+> Note: From cert-manager `v1.2.0` onward, the minimum supported version of
+> Kubernetes is `v1.16.0`. Users still running Kubernetes `v1.15` or below should
+> upgrade to a supported version before installing cert-manager.
+
+> **Warning**: You should not install multiple instances of cert-manager on a single
+> cluster. This will lead to undefined behavior and you may be banned from
+> providers such as Let's Encrypt.
+
+## Installing with Helm
+
+> **Note**: cert-manager should never be embedded as a sub-chart into other Helm charts.
+> cert-manager manages non-namespaced resources in your cluster and should only be installed once.
+
+### Prerequisites
+
+- Helm v3 installed
+
+### Steps
+
+In order to install the Helm chart, you must follow these steps:
+
+#### 1. Add the Jetstack Helm repository:
+
+> **Warning**: It is important that this repository is used to install
+> cert-manager. The version residing in the [Helm stable repository](https://github.com/helm/charts/tree/master/stable/cert-manager) is
+> *deprecated* and should *NOT* be used.
```

Comment:
suggestion: I don't think we should link to a deprecated repository; we should just encourage people to use our repo. I also don't think this needs to be highlighted as a warning.

```suggestion
This repository is the only supported source of cert-manager charts. There are some other mirrors and copies across the internet, but those are entirely unofficial and could present a security risk.

Notably, the "Helm stable repository" version of cert-manager is deprecated and should not be used.
```